components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/tool-editor/views/tool-editor.js

```diff
@@ -82,9 +82,11 @@ define(['require', 'jquery', 'backbone', 'lodash', 'log', './design', "./source"
 
                     /* Start Debug Related Stuff */
                     var debugConfOpts = {
-                        debuggerInstance : self._sourceView.getDebugger(),
-                        editorInstance : self._sourceView.getEditor()
-                    }
+                        debugger_instance : self._sourceView.getDebugger(),
+                        editorInstance : self._sourceView.getEditor(),
+                        option : self.options.application.config.debugger_instance
```

Comment:
Actually we added this when we tried to add hover to the debugger buttons. All commands and button attributes which we defined in under debugger_instance in _index.html_ will assigned to **option**.  Since we still didn't implement it we can remove it. Shall I remove it?